src/main/java/org/aion/wallet/ui/components/SendController.java

```diff
@@ -26,12 +26,19 @@
 
 import java.math.BigInteger;
 import java.net.URL;
+import java.util.Optional;
 import java.util.ResourceBundle;
 
 public class SendController extends AbstractController {
 
     private static final Logger log = WalletLoggerFactory.getLogger(LogEnum.WLT.name());
 
+    private static final String ERROR_LABEL = "error-label";
```

Comment:
create an enum for the possible statuses of a transaction

Reply:
This is actually a badly named css style class